pkg/apis/pipeline/v1/pipeline_defaults_test.go

```diff
@@ -45,6 +45,42 @@ func TestPipelineSpec_SetDefaults(t *testing.T) {
 		desc: "empty pipelineSpec must not change after setting defaults",
 		ps:   &v1.PipelineSpec{},
 		want: &v1.PipelineSpec{},
+	}, {
```

Comment:
can we move these tests to its own file in the same package? Will reduce changes to merge conflict

Reply:
sure!